specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManager.json

```diff
@@ -507,6 +507,13 @@
                 "type": "string"
               },
               "description": "List of subscriptions."
+            },
+            "crossTenantScopes": {
+              "type": "array",
```

Comment:
Please get sign off from breaking changes

Reply:
This is not a breaking change. It is just the addition of a new property